src/ViewModel/Magewire.php

```diff
@@ -67,7 +67,12 @@ public function isBeforeTwoFourZero(): bool
 
     public function getPostRoute(): string
     {
-        return $this->isBeforeTwoFourZero() ? '/magewire/vintage' : '/magewire/post';
+        $postRoute = '/magewire/post';
+        if (($this->getSystemName() === 'Magento') && ($this->isBeforeTwoFourZero() === true)) {
```

Comment:
Can you change this from:
`if(($this->getSystemName() === 'Magento') && ($this->isBeforeTwoFourZero() === true)) {}`

To

`if($this->getSystemName() === 'Magento' && $this->isBeforeTwoFourZero()) {}`

I think it's a bit cleaner.